python/lsst/ctrl/bps/parsl/sites/work_queue.py

```diff
@@ -0,0 +1,140 @@
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
+
+from parsl.executors import WorkQueueExecutor
+from parsl.executors.base import ParslExecutor
+from parsl.launchers import SrunLauncher
+from parsl.providers import LocalProvider
+from parsl.providers.provider_base import ExecutionProvider
+
+from ..configuration import get_bps_config_value
+from ..site import SiteConfig
+
+if TYPE_CHECKING:
+    from ..job import ParslJob
+
+__all__ = ("WorkQueue", "SlurmWorkQueue")
+
+
+Kwargs = Dict[str, Any]
```

Comment:
Unused, I think. Please check that everything passes linting and you've run `black`.